tests/models/gpt_neox/test_modeling_gpt_neox.py

```diff
@@ -314,9 +314,10 @@ def test_lm_generate_gptneox(self):
             model.to(torch_device)
 
             inputs = tokenizer("My favorite food is", return_tensors="pt").to(torch_device)
-            expected_output = (
-                "My favorite food is the chicken and rice.\n\nI love to cook and bake. I love to cook and bake"
-            )
+            # On April 4th 2023, files on `EleutherAI/pythia-410m-deduped` are updated (and the previous versions
+            # completely disappear in the commit history). Unfortunately, the output of this `inputs` doesn't make a lot
+            # of sense.
```

Comment:
This could be a bit terser. Just a suggestion, not a strong opinion :) 

```suggestion
            # 2023-04-04 weights updated, resulting in poor outputs. See: https://github.com/huggingface/transformers/pull/24193
```